packer/install-dependencies.sh

```diff
@@ -394,25 +381,13 @@ for KMS_PLUGIN_VERSION in ${KMS_PLUGIN_VERSIONS}; do
     echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
 done
 
-FLANNEL_VERSIONS="
-0.10.0
-0.8.0
-"
-for FLANNEL_VERSION in ${FLANNEL_VERSIONS}; do
-    CONTAINER_IMAGE="quay.io/coreos/flannel:v${FLANNEL_VERSION}-amd64"
-    pullContainerImage "docker" ${CONTAINER_IMAGE}
-    echo "  - ${CONTAINER_IMAGE}" >> ${VHD_LOGS_FILEPATH}
-done
-
 pullContainerImage "docker" "busybox"
 echo "  - busybox" >> ${VHD_LOGS_FILEPATH}
 
 K8S_VERSIONS="
 1.17.0
 1.16.4
```

Comment:
We've moved to all to MCR, I don't think we need the hyperkube images from gcr now.

Reply:
@gossion, can you confirm?

Reply:
that is right. We have moved all to MCR. 
So this VHD is only for AKS, right? if that is true, we should remove all gcr, dockerhub, quay images - per our egress policy, no traffic should go to gcr by default.

Reply:
in fact we need to pull these from mcr as we're now seeing error due to below image missing
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.16.4
mcr.microsoft.com/oss/kubernetes/hyperkube:v1.15.7

Reply:
@gossion yes, this is AKS "private" VHD. One edge case is the scale up for VMAS, which would still refer to old setting(from the pod, say kubeproxy), and when it uses the new VHD, it will need to pull images from GCR. But for VMAS scaling operation, we only need to support the 1.13+ vesrions.